src/stories/tasks/TasksTable.stories.mdx

```diff
@@ -25,8 +33,20 @@ A description of the **Task** that links to its details page.
 
 **Status**
 
-Displays the **Task**'s status as `Planned`, `In Progress`, `Complete`,
-`Changes Requested`, or `Approved`.
+Displays the **Task**'s status based on the work of the `Developer` and the
+`Tester`.
+
+- A **Task** begins with a status of `Planned` until a `Scratch Org` has been
+  created, which updates the status to `In Progress`.
+- It remains `In Progress` until a `Tester` submits a review, at which point the
+  status becomes `Changes Requested` or `Approved`.
+- If changes are made or retrieved (even after review), it goes back to
+  `In Progress` and will be updated once another review from the `Tester` is
+  submitted.
+- The review status is only shown if the review was submitted on the _most
+  recent commit_ of the **Task** branch.
```

Comment:
```suggestion
- The review status is only shown if the review was submitted on the _most
  recent commit_ of the **Task** branch. Otherwise it is `In Progress`.
```